assembl/static2/js/app/components/debate/thread/nuggets.jsx

```diff
@@ -45,15 +46,59 @@ class Nuggets extends React.Component {
   }
 
   render() {
-    const { extracts } = this.props;
+    const { extracts, isHarvesting } = this.props;
     const { top } = this.state;
+    if (isHarvesting) {
+      if (Array.isArray(extracts) && extracts.length > 0) {
+        return (
+          <div
+            ref={(node) => {
+              this.node = node;
+            }}
+            className="extracts extracts--is-harvesting"
+            style={Nuggets.topToStyle(top)}
+          >
+            {extracts.map(extract => (
+              <div key={extract.id} className="extract">
+                <div className="extract-header">
+                  <div className="extract-harvester">
+                    <div className="extract-harvester-left">
+                      <div className="extract-harvester-avatar">
+                        <img
+                          src="/static2/img/icons/avatar.png?TODO"
```

Comment:
Haha interesting TODO. You should put this in the constants file, and refer to it here IF the profile query in the apollo store does NOT have a profile URL. You are repeating this path more than once.

Reply:
This story limits itself to showing an extract on the left of the message in a different manner when the harvesting mode is on. Getting data about harvester and using it is part of a subsequent story, as well as having buttons in this box that do things.